Classes/Notifications/NotificationsViewController.swift

```diff
@@ -119,6 +123,8 @@ NotificationNextPageSectionControllerDelegate {
         )
         feed.finishLoading(dismissRefresh: dismissRefresh, animated: animated)
         updateMarkAllEnabled()
+        
+        showDetailViewController(SplitPlaceholderViewController(), sender: self)
```

Comment:
I was suspect of this, going to remove since it pushes a placeholder VC on iPhone when starting in portrait.